spec/NewTwitchApi/Resources/UsersApiSpec.php

```diff
@@ -10,98 +10,110 @@
 
 class UsersApiSpec extends ObjectBehavior
 {
-    function let(Client $guzzleClient)
+    public function let(Client $guzzleClient)
     {
         $this->beConstructedWith($guzzleClient);
     }
 
-    function it_should_get_users_by_ids(Client $guzzleClient, Response $response)
+    public function it_should_get_users_by_ids(Client $guzzleClient, Response $response)
     {
         $guzzleClient->send(new Request('GET', 'users?id=12345&id=98765', ['Authorization' => 'Bearer TEST_TOKEN']))->willReturn($response);
-        $this->getUsers("TEST_TOKEN", ['12345','98765'])->shouldBeAnInstanceOf(ResponseInterface::class);
+        $this->getUsers('TEST_TOKEN', ['12345', '98765'])->shouldBeAnInstanceOf(ResponseInterface::class);
```

Comment:
Thanks for catching these! The specs aren't checked by PHPSpec because of the explanation in my other comment about specs not being considered "code", so sometimes things like this slip through into the specs. 

Perhaps I can find a way to run php-cs-fixer on specs with their own set of rules, so help with this in the future. :thinking: